userspace/falco/app/actions/process_events.cpp

```diff
@@ -370,16 +370,17 @@ static void process_inspector_events(
 		{
 			sdropmgr.print_stats();
 		}
+
+		if (sync)
```

Comment:
not clear to me why we need this :thinking: before the patch, we managed the sync even in case of exception while now in case of exception, we don't call `sync->finish()`. Could you explain the patch?

Reply:
Wrong implementation, I wanted to surround the sync in its own try block since it's a method that can throw.